.woodpecker/docker.yml

```diff
@@ -3,7 +3,6 @@ when:
   - event: push
     branch:
       - ${CI_REPO_DEFAULT_BRANCH}
-      - release/*
```

Comment:
why remove release branches - this will prohibit releasing from them patch versions

Reply:
thats a point